scipy/optimize/_linprog_simplex.py

```diff
@@ -229,7 +229,7 @@ def _apply_pivot(T, basis, pivrow, pivcol, tol=1e-12):
 
```

Comment:
Do you mind adding a stacklevel arg for this one while we are at it?

Reply:
You mean on the warnings?

Reply:
yes indeed

Reply:
@ilayn I haven't used stacklevel much before - what's the advantage of adding them? And what would you recommend for the default value?

Reply:
@ilayn wrote in #10026:
>The idea of the stacklevel is that you get the warning from the usercode line n that actually triggered this warning and not from some internal code that is difficult for the user to hunt down the reason. Imagine having a 200 line code and you see some warnings popping up. You would like to know which line actually triggers it. But if the warnings point to some internal function you wouldn't know hence the stacklevel is used to climb up in terms of the scope. @pv taught me about this but the mistakes are mine.

See example there.